### PR TITLE
Readability Changes

### PR DESCRIPTION
**Pull request by vitorpamplona**
- changed the order of some sections and sentences, reduced title sizes, changed the hierarchy level of sections
- and added options for PubKeys in the DNS records and as well as downloadable files.

### CODE REVIEW

File: SPECIFICATION.md [hunk 1]
```diff
@@ -4,43 +4,72 @@ Authors: [Justin Dosse](mailto:justin.dossey@pathcheck.org) and [Vitor Pamplona]
...
+1. the **version**
+1. the **payload** itself
+1. the reference to a **public key**
+1. and a cryptographic **signature** of the payload
+
+The **type** field defines the payload type (`COUPON`, `PASSKEY`, `BADGE` or `STATUS`) and the **version** is a **NUMERIC** field defining the version of the type of certificate. The **payload** section contains the information related to the certificate itself. The cryptographic signature is a SHA256 signature in hexadecimal form, calculated using the private ECDSA key of the **ISSUER**. 
+
+The reference to the **public** key can be: 
+1. a URI to a DNS TXT Record containing the key for download.
+1. a URL address to download keys from.
```
Comment (justindossey) [inline SPECIFICATION.md:47]: I'm still not a fan of trying to fit a URL into keyId

File: SPECIFICATION.md [hunk 2]
```diff
@@ -4,43 +4,72 @@ Authors: [Justin Dosse](mailto:justin.dossey@pathcheck.org) and [Vitor Pamplona]
...
+1. the **type** of certificate
+1. the **version**
+1. the **payload** itself
+1. the reference to a **public key**
+1. and a cryptographic **signature** of the payload
+
+The **type** field defines the payload type (`COUPON`, `PASSKEY`, `BADGE` or `STATUS`) and the **version** is a **NUMERIC** field defining the version of the type of certificate. The **payload** section contains the information related to the certificate itself. The cryptographic signature is a SHA256 signature in hexadecimal form, calculated using the private ECDSA key of the **ISSUER**. 
+
+The reference to the **public** key can be: 
+1. a URI to a DNS TXT Record containing the key for download.
```
Comment (justindossey) [inline SPECIFICATION.md:46]: I think FQDN makes more sense here than URI

**Review by justindossey — CHANGES_REQUESTED** [review]

File: SPECIFICATION.md (hunk 2 above)
Comment (vitorpamplona) [reply SPECIFICATION.md]: Nice! I agree.

File: SPECIFICATION.md (hunk 1 above)
Comment (vitorpamplona) [reply SPECIFICATION.md:47]: I am keeping it for now because it's very useful to test. But I also don't like it.

**Pull request merged**